templates/ts/graph-connector/src/custom/getExternalItemFromItem.ts

```diff
@@ -2,6 +2,15 @@ import { Item } from "../models/Item";
 import { ExternalConnectors } from "@microsoft/microsoft-graph-types";
 import { getAclFromITem } from "./getAclFromItem";
 
+// [Customization point]
```

Comment:
Add a link to the Graph docs where we can understand the shape of an External Item